host-program/src/bin/add-go-prove.rs

```
@@ -81,21 +82,45 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         seg_size: seg_size2,
         execute_only: execute_only2,
     };
+
+    match fs::create_dir_all(&output_dir) {
+        Ok(_) => log::info!("{} created successfully.", &output_dir),
+        Err(e) => {
+            log::info!("Failed to create directory {}, err: {}", &output_dir, e);
+            return Ok(());
```

Why we return ok()? if we want to ignore the error, just use let _ = fs::create_dir_all(&output_dir).

---

Do the same for other file operation.